openproblems/tasks/_batch_integration/batch_integration_graph/api.py

```diff
@@ -0,0 +1,34 @@
+from .datasets.immune import immune_batch
+
+
+def check_dataset(adata):
+    """Check that dataset output fits expected API."""
+
+    assert "X_uni" in adata.obsm
```

Comment:
I assume this is old code and should now be a test for `adata.layers['counts']`?

Reply:
Otherwise, this is not stated in the README and should be added... it seems to be used as a representation below for the `sample_method` function.